contracts/ics721/src/contract_test.rs

```diff
@@ -1,666 +0,0 @@
-#[cfg(test)]
```

Comment:
We are deleting this entire file
But there is a lot of good checking of response types and expected data here
Still going to need tests like these that are very explicit

Reply:
Agree. I've actually moved over one or two tests from that file more-or-less verbatim to the [new file](https://github.com/public-awesome/ics721/blob/zeke/ics721/contracts/cw-ics721-bridge/src/ibc_tests.rs). Think the coverage there should be about the same when you include `integration_tests.rs` which checks some state things being checked in in `ibc_tests` before.

Reply:
> Agree. I've actually moved over one or two tests from that file more-or-less verbatim to the [new file](https://github.com/public-awesome/ics721/blob/zeke/ics721/contracts/cw-ics721-bridge/src/ibc_tests.rs). Think the coverage there should be about the same when you include `integration_tests.rs` which checks some state things being checked in in `ibc_tests` before.

Cool thank you 👍